tests/test_integration.py

```diff
@@ -76,14 +77,37 @@ def test_benchmark_models(
     state_space_functions,
     load_example_model,
 ):
-    params, options = load_example_model(f"{model}")
-    options["n_exog_states"] = 1
+    options = {}
+    params, _options = load_example_model(f"{model}")
+    _options["n_exog_states"] = 1
+
+    options["model_params"] = _options
+    options.update(
+        {
+            # "exogenous_processes": {
+            #     "exog": _return_one,
+            # },
+            "state_variables": {
+                "endogenous": {
+                    "period": jnp.arange(2),
+                    "lagged_choice": [0, 1],
+                },
+                # "exogenous": {"exog_state": []},
+                "choice": [0, 1],
+            },
+            # "model_params": {},
+        }
+    )
 
-    exog_savings_grid = jnp.linspace(0, options["max_wealth"], options["n_grid_points"])
+    exog_savings_grid = jnp.linspace(
+        0,
+        options["model_params"]["max_wealth"],
+        options["model_params"]["n_grid_points"],
+    )
 
-    state_space, map_state_to_index = create_state_space(options)
+    state_space, map_state_to_index = create_state_space(options["model_params"])
     state_choice_space, *_ = create_state_choice_space(
-        options,
+        options["model_params"],
```

Comment:
All information here should be contained in the "state_space" part. See comment before.